### DISCUSSION TITLE
Map Lurker on kirack mission

### DISCUSSION THREAD

**Discussion by shakaah** [C1rdec/PoeLurker#371] [Q&A] [answered]
Hi, and a big thanks for this mod
I've a question, is it possible to use the "Map Lurker" on kirack inventory / kirack mission ?
![image](https://user-images.githubusercontent.com/20798484/214396421-bd97f878-2f6b-45f4-8826-8c865a13ea1f.png)

If not, is it planed for futur update?
Thanks a lot for your work
Shakaah

**Comment by C1rdec** [comment, marked as answer]
This is not possible sadly.... when you you perform the CTRL + C in that screen the map details is not copy to the clipboard.

**Reply by shakaah** [reply]
ok, thanks for the answer
